chill-bijection/src/main/scala/com/twitter/chill/ExternalizerCodec.scala

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2012 Twitter, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package com.twitter.chill
+
+import _root_.java.io.{
+  ByteArrayOutputStream,
+  ByteArrayInputStream,
+  ObjectInputStream,
+  ObjectOutputStream
+}
+
+import com.twitter.bijection.Injection
+import scala.util.Try
+
+/**
+ * A Injection to serialize an externalizer to bytes
+ */
+object ExternalizerCodec {
+  implicit def apply[T]: ExternalizerCodec[T] = new ExternalizerCodec[T]
+}
+
+class ExternalizerCodec[T] extends Injection[Externalizer[T], Array[Byte]] {
+  def apply(extern: Externalizer[T]): Array[Byte] = {
+    val baos = new ByteArrayOutputStream()
+    val oos = new ObjectOutputStream(baos)
+    oos.writeObject(extern)
+    baos.toByteArray
+  }
+  def invert(bytes: Array[Byte]): Try[Externalizer[T]] = Try {
```

Comment:
still not totally safe, because if you deserialize `Externalizer[String]` or `Externalizer[Int]` both will not throw until you call .get and it fails to cast to `String` or `Int`.

We could solve one layer deep, but then it could also fail for collections: `Externalizer[List[T]]`.

So, I'd rather just add a comment that this is unsafe and can result in class cast exceptions at runtime.